### PR TITLE
fix up intro category

### PR DESCRIPTION
**Pull request by ix0rai**
I don't believe that we should be justifying quilt's existence on the first page of the wiki. removes all visible TODOs in the intro section

---
See preview on Cloudflare Pages: https://preview-98.developer-wiki.pages.dev

### CODE REVIEW

File: wiki/introduction/setting-up/en.md [hunk 1]
```diff
@@ -207,4 +209,4 @@ Finally, reload the Gradle project by pressing this button in the Gradle menu:
 </picture>
 
 Once you're done with all of these steps, your mod is ready to be worked on!
-You can start by [Creating your First Item](../items/first-item)
+You can start by adding content via the [Creating your First Item](../items/first-item) and [Creating your First Block](../items/first-block) tutorials.
```
Comment (woodiertexas) [inline wiki/introduction/setting-up/en.md:212]: yes, creating content via the "creating your first item" :yeefpineapple:

Comment (ix0rai) [reply wiki/introduction/setting-up/en.md:212]: it makes grammatical sense

Comment (woodiertexas) [reply wiki/introduction/setting-up/en.md]: i'd probably add quotes tbh

`"Creating your First Item"` and `"Creating your First Block"`

Comment (ix0rai) [reply wiki/introduction/setting-up/en.md]: fine

**Comment by ix0rai** [comment]
remember when I said 2 days five days ago. set yourself reminders kids

**Pull request merged**